python/ray/train/huggingface/utils.py

```diff
@@ -19,6 +21,42 @@
     from ray.data.preprocessor import Preprocessor
 
 
+@PublicAPI(stability="alpha")
+def to_air_checkpoint(
+    model: Union[transformers.modeling_utils.PreTrainedModel, torch.nn.Module],
+    tokenizer: Optional[transformers.PreTrainedTokenizer] = None,
+    preprocessor: Optional["Preprocessor"] = None,
+    path: Optional[str] = None,
+) -> Checkpoint:
+    """Convert a pretrained Transformers model to AIR checkpoint for serve or inference.
+
+    Args:
+        model: Either a ``transformers.PreTrainedModel``, or a trained PyTorch model.
+        tokenizer: Tokenizer to use.
```

Comment:
Is there any more information we can provide on the tokenizer/how it will be used?